torch_geometric/datasets/brca_tgca.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import Callable, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+import torch
+
+from torch_geometric.data import (
+    Data,
+    InMemoryDataset,
+    download_url,
+    extract_zip,
+)
+
+
+class BrcaTcga(InMemoryDataset):
+    r"""The breast cancer (BRCA TCGA) dataset from `cBioPortal
+    <https://www.cbioportal.org>`_ and the biological network for node
+    connections from `Pathway Commons <https://www.pathwaycommons.org>`_.
+    The dataset contains the gene features of each patient in graph_features
+    and the overall survival time (in months) of each patient,
+    which are the labels.
```

Comment:
Could you check the docstring change as I've just removed some redundant sentences? Also, it'd be ncie if you could describe what nodes and edges represent to help new users understand this dataset. (example: https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.datasets.KarateClub.html#torch_geometric.datasets.KarateClub)